javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -53,6 +54,7 @@
 
     public static final int STRICTFP = Modifier.STRICT;
 
+    @Deprecated
```

Comment:
not sure if I want to suggest that or not - but according to DRY, this should be delegated to the new (or an extracted) method

Usually I would want that, but it is very unlikely that this code will ever be touched again...